.env.development

```diff
@@ -1 +1,2 @@
-VITE_BACKEND_URL=http://localhost:3000/api
\ No newline at end of file
+VITE_BACKEND_URL=http://localhost:3000/api
+VITE_CLIENT_ID=<google-client-id>
```

Comment:
No me convence tener el client id de google aca, al final si alguien ve el repo, puede obtener el codigo y pasan cosas chistosas, revisa como poder usarlo desde un .env o como protegerlo para que no se filtre

Reply:
greetings
estuve revisando y el google_client_id es público, no hay necesidad de ocultarlo. El complicado es el client_secret, ese sí no se tiene que filtrar por ningún motivo jajaja

por cierto, ese archivo es un .env, así que no debería estar nunca el client id en el repo

https://stackoverflow.com/questions/34833820/do-we-need-to-hide-the-google-oauth-client-id
https://www.googlecloudcommunity.com/gc/Apigee/Client-Id-vs-Client-Secret/m-p/13501

de hecho, me fijé y cuando haces click en Iniciar sesión con google en el login del blackshark, te aparece una ventana emergente. En la url de esa ventana, aparece el client_id

en todo caso, algo que se puede hacer es hacer un endpoint en el backend que devuelva el client_id, y usarlo desde ahí. Luego, cuando se deployee el backend, configurarlo para que solo acepte requests del dominio del frontend

ahí avísame que hacer jajajaj

Reply:
Si no es necesario ocultarlo, dejemoslo tal cual esta ahora, lo que no entendi bien es a que te refieres con lo del .env, en este caso, los 2 .env que existen son meramente para poder diferenciar las urls de los backends cuando se sube a prod, si lo quieres cambiar me avisas, para asi poder revisar como lo podemos dejar seteado ajajajaja

Lo del client id dejalo como esta ahora, ahi cualquier cosa me avisas y nos juntamos en ds por ultimo para verlo